package.json

```diff
@@ -1,17 +1,21 @@
 {
   "name": "dashboard-app",
-  "version": "1.2.1",
+  "version": "1.3.0",
+  "license": "Apache-2.0",
   "private": true,
   "dependencies": {
+    "@catena-x/portal-shared-components": "2.1.3",
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
     "@mui/icons-material": "5.10.6",
-    "@mui/material": "5.10.7",
+    "@mui/x-data-grid": "5.17.26",
+    "@mui/types": "7.2.5",
     "@testing-library/jest-dom": "^6.1.4",
     "@testing-library/react": "14.1.0",
     "@testing-library/user-event": "^14.5.1",
     "axios": "1.6.1",
-    "cx-portal-shared-components": "^0.4.5",
+    "buffer": "^6.0.3",
+    "cx-portal-shared-components": "0.4.5",
```

Comment:
why do you need the deprecated lib as well?
@catena-x/portal-shared-components should be the leading one.